examples/language-sdk-instrumentation/rust/README.md

```diff
@@ -1,2 +1,148 @@
+
+## Where's the performance bottleneck?
+
+![Image](https://github.com/user-attachments/assets/d4b0f85d-cc8d-4058-b019-1c5198849676)
+
+The first step when analyzing a profile outputted from your application, is to take note of the _largest node_ which is
+where your application is spending the most resources. In this case, it happens to be the `order_car` function.
+
+The benefit of using the Pyroscope package, is that now that we can investigate further as to _why_ the `order_car()`
+function is problematic. Tagging both `region` and `vehicle` allows us to test two good hypotheses:
+
+- Something is wrong with the `/car` endpoint code
+- Something is wrong with one of our regions
+
+To analyze this we can select one or more tags on the "Labels" page:
+
+![Image](https://github.com/user-attachments/assets/3e5cb3ac-609e-493a-ae4d-248de150a33b)
+
+## Narrowing in on the Issue Using Tags
+
+Knowing there is an issue with the `order_car` function we automatically select that tag. Then, after inspecting
+multiple `region` tags, it becomes clear by looking at the timeline that there is an issue with the `eu-north` region,
+where it alternates between high-cpu times and low-cpu times.
+
+We can also see that the `mutex_lock()` function is consuming almost 70% of CPU resources during this time period.
+
+![Image](https://github.com/user-attachments/assets/12fc0912-8b65-4c24-9284-b0aa1eef45ba)
+
+## Visualizing Diff Between Two Flame graphs
+
+While the difference _in this case_ is stark enough to see in the comparison view, sometimes the diff between the two
+flame graphs is better visualized with them overlayed over each other. Without changing any parameters, we can simply
+select the diff view tab and see the difference represented in a color-coded diff flame graph.
+
+![Image](https://github.com/user-attachments/assets/97f6e51c-4211-4a0a-8f11-d2ee0402e396)
+
+### More use cases
+
+We have been beta testing this feature with several different companies and some of the ways that we've seen companies
+tag their performance data:
+
+- Tagging Kubernetes attributes
+- Tagging controllers
+- Tagging regions
+- Tagging jobs from a queue
+- Tagging commits
+- Tagging staging / production environments
+- Tagging different parts of their testing suites
+- Etc...
+
+### Future Roadmap
+
+We would love for you to try out this example and see what ways you can adapt this to your Rust application. Continuous
+profiling has become an increasingly popular tool for the monitoring and debugging of performance issues (arguably the
+fourth pillar of observability).
+
+We'd love to continue to improve our Rust integrations, and so we would love to hear what features _you would like to
```

Comment:
Should we include a suggested method for contacting us?

Reply:
idk, This was copypasted from somewhere else. There used to be a twitter handle on one of screenshots. If we need to suggest a method, it should be done everywhere